spa/src/pages/ConfigSteps/OrgsContainer/index.tsx

```diff
@@ -49,8 +56,54 @@ const OrganizationsList = ({
 	resetCallback: (args: boolean) => void;
 	connectingOrg: (org: GitHubInstallationType) => void;
 }) => {
-	const [clickedOrg, setClickedOrg] = useState<GitHubInstallationType | undefined>(undefined);
-	const canConnect = (org: GitHubInstallationType) => !org.requiresSsoLogin && !org.isIPBlocked && org.isAdmin;
+	const [isScrolledToBottom, setIsScrolledToBottom] = useState(false);
+	const [isListScrollable, setIsListScrollable] = useState(false);
+	const contentRef = useRef<HTMLDivElement | null>(null);
+	const hasScrolledYetRef = useRef(false);
+	const [clickedOrg, setClickedOrg] = useState<
+		GitHubInstallationType | undefined
+	>(undefined);
+
+	const canConnect = (org: GitHubInstallationType) =>
+		!org.requiresSsoLogin && !org.isIPBlocked && org.isAdmin;
+
+	const checkIsListScrollable = () => {
+		let isListScrollable = false;
+		const content = contentRef.current;
+		if (content) {
+			const max = parseInt(window.getComputedStyle(content).maxHeight);
+			const size = content.scrollHeight;
+			isListScrollable =  (size - 15) > max;
+
+		}
+		setIsListScrollable(isListScrollable);
+	};
+
+	const handleScroll = ()=> {
+		const content = contentRef.current;
+		hasScrolledYetRef.current = true;
+		if (content) {
+			const scrollTop = content.scrollTop;
```

Comment:
\nit
Same here, this can also be moved to the another util method which simply checks if the scrollbar has reached the bottom of the DOM or not.